pkg/spec/config_linux.go

```diff
@@ -32,6 +32,11 @@ func Device(d *configs.Device) spec.LinuxDevice {
 	}
 }
 
+// DevicesFromPath is an exported wrapper for devicesFromPath
+func DevicesFromPath(g *generate.Generator, devicePath string) error {
```

Comment:
Can we just rename devicesFromPath instead? I don't think we need separate exported/not exported versions